import-automation/executor/app/executor/cloud_run_simple_import.py

```diff
@@ -0,0 +1,217 @@
+        file_util.file_copy(config_file, gcs_config_file)
+    return gcs_config_file
+
+
+def cloud_run_simple_import_job(
+    import_spec: str,
+    config_file: str,
+    env: dict = {},
+    version: str = '',
+    job_id: str = '',
+    project_id: str = _DEFAULT_PROJECT,
+    location: str = _DEFAULT_LOCATION,
+    image: str = _DEFAULT_SIMPLE_IMAGE,
+) -> str:
+    """Create and run a Cloud Run job for simple a import.
+
+  Args:
+    import_name: name of the import.
+      Folder for output will have the same name.
+    config_file: json file with config for simple import run.
+      For an example, see simple_imports/sample/import_config.json
+    env: dictionary of environment variables for the job.
+      Additional variabled for CONFIG_FILE and OUTPUT_DIR are added.
+    version: a dated version string for file path for output folder.
+    project_id: Google project id for the cloud run. The compute engine service
+      account for the project should have access to the GCS folder for output.
+    location: Region for the cloud run instance.
+    image: container image URL for the simple-importer.
+
+  Returns:
+    Output directory with the script outputs.
+  """
+    if not version:
+        version = _get_time_version()
+    import_name = import_spec.get('import_name', '')
+    # Copy the config file to GCS versioned folder.
+    # Cloud run can only access GCS for config files, input and output.
+    gcs_config = get_simple_import_gcs_config(config_file,
+                                              import_name,
+                                              version,
+                                              copy_file=True)
+    gcs_output_dir = os.path.dirname(gcs_config)
+
+    # Setup environment variables for simple import run.
+    # Settings for each import run have to be passed in through ENV variables.
+    env_vars = {
+        'CONFIG_FILE': gcs_config,
+        'OUTPUT_DIR': gcs_output_dir,
+    }
+    if env:
+        env_var.update(env)
+
+    # Generate job-id from config_file path if not specified.
+    if not job_id:
+        job_id = get_simple_import_job_id(import_spec, config_file)
+
+    # Create the job for the config.
+    # An existing job is updated with new env variables for versioned output
+    logging.info(
+        f'Creating simple import cloud run {project_id}:{job_id} for'
+        f' {config_file} with output: {gcs_output_dir}, env: {env_vars}')
+    job = cloud_run.create_or_update_cloud_run_job(project_id, location, job_id,
+                                                   image, env_vars)
+    if not job:
+        logging.error(
+            f'Failed to create cloud run job {job_id} for {config_file}')
+        return None
+
+    # Execute the cloud run job.
+    logging.info(f'Running {project_id}:{job_id} for {config_file} with output:'
+                 f' {gcs_output_dir}')
+    job = cloud_run.execute_cloud_run_job(project_id, location, job_id)
+    if not job:
```

Comment:
Can you add a TODO to look at `report.json` for final status?

Reply:
Added TODO